Fix device base comment and explanation at working_with_time_series.md (#1477)
NREL-Sienna/PowerSystems.jl: docs/src/tutorials/working_with_time_series.md

Notice that we assigned the
[`get_max_active_power`](@ref get_max_active_power(value::PowerLoad)) *function*
to scale the time series, rather than a value, making the time series reusable for multiple

> components or multiple fields in a component. Note that the values are per-unitized using

Reviewer: ```suggestion
components or multiple fields in a component. Note that the values are normalized using
```

> each device’s max_active_power, not the system-wide base_power.

Reviewer: ```suggestion
each device’s `max_active_power` parameter, not the system-wide `base_power`.
```

    
    Notice that each load now has two references to `max_active_power`. This is intentional.
    There is the parameter, `max_active_power`, which is  the

>     maximum demand of each load at any time (10 MW and 30 MW). There is also

Reviewer: ```suggestion
    maximum demand of each load at any time (10 MW or 30 MW). There is also
```
